src/misc/generate-docs.ts

```diff
@@ -12,8 +12,8 @@ function updateUsage(
   actionReference: string,
   actionYamlPath = 'action.yml',
   readmePath = 'README.md',
-  startToken = '<!-- start usage -->',
-  endToken = '<!-- end usage -->'
+  startToken = '<!-- start usage -->\n',
```

Comment:
This is the only functional/manual change so it stays in sync with the prettier formatted readme and still finds the token running `npm run build`